abstract_commands.xml

```diff
@@ -2,13 +2,13 @@
     <register name="Access Register">
         \begin{steps}{Perform the following sequence of operations:}
         \item Halt the hart, if \Fprehalt is set. If the hart is already
-            handled, the entire command fails.
+            halted, the entire command fails.
         \item Execute the Program Buffer, if \Fpreexec is set.
         \item Copy data from the register specified by \Fregno into {\tt data}, if \Fwrite is clear.
         \item Copy data from {\tt data} into the register specified by \Fregno, if \Fwrite is set.
         \item Execute the Program Buffer, if \Fpostexec is set.
-        \item Resume the hart, if \Fpostresume is set. If the hart is already
-            running, the entire command fails.
+        \item Resume the hart, if \Fpostresume is set. If the hart is already 
+        running and \Fprehalt is not set, the entire command fails.
```

Comment:
No, it doesn't make sense. The hart may already be running, but if you do prehalt, then it will be halted. So it is OK if you do prehalt & postresume together. Or are you saying you would check only at the end of the command's execution? That's weird when placed next to the bit's description, and it seems you'd rather decide at the beginning whether this command is legal (s.t. the only failures you get while running it are exceptions).